plugins/notification/src/index.ts

```diff
@@ -95,6 +95,16 @@ export interface NotificationTemplate {
   subjectTemplate: string
 }
 
+/**
+ * @public
+ */
+export interface NotificationPresentation {
+  title: IntlString
+  body: IntlString
+  intlParams: Record<string, string | number>
+  intlParamsNotLocalized?: Record<string, IntlString>
```

Comment:
Should sender be a part of it?

Reply:
Or Icon if it is a service send.

Reply:
senderName for some notification is a part of intlParams. sender itself stored in modifiedBy of transaction as was before. Again we decided to continue only with adding localization properties to current implementation.